react/game/src/utils/auth.js

```diff
@@ -0,0 +1,11 @@
+import { AUTH_SESSION_NAME } from '@constants/localStorage';
+import { service as localStorageService } from '@services/localStorageService';
+
+const loadAuthState = () => {
+  const session = localStorageService.get(AUTH_SESSION_NAME);
+  return session || null;
+};
+
+const saveAuthState = value => value && localStorageService.set(AUTH_SESSION_NAME, value);
+
+export { loadAuthState, saveAuthState };
```

Comment:
I'd move all this to `localStorageService` file, and I'd convert `get`, `set` and `delete` to private functions